spring-xsuaa/pom.xml

```diff
@@ -146,6 +146,10 @@
 			<artifactId>reactor-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
for which purpose this is required?

Reply:
@nenaraab To parse inputstream into Map in [this](https://github.com/SAP/cloud-security-xsuaa-integration/blob/7e3e1a880879ba7a318fc47ff39469698d55e027/spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/XsuaaServicesParser.java#L69) constructor